pkg/consts/views.go

```diff
@@ -13,6 +13,8 @@ const IndexViewsVersion int = 16
 // properly.
 var GlobalIndexes = []*mango.Index{
 	mango.IndexOnFields(Instances, "by-domain", []string{"domain"}),
+
+	mango.IndexOnFields(Exports, "by-domain", []string{"domain", "created_at"}),
```

Comment:
Don't forget to increase `IndexViewsVersion`

Reply:
Done 62a7d65bc013d0b8c9d068830b5ae8ed66c6057f.